### PR TITLE
Fix T857353: Grids - Describe the use of the pageSize for all scrolling modes

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Review by AlexanderKovalenko — APPROVED** [review]

File: api-reference/10 UI Widgets/GridBase/1 Configuration/paging/paging.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ type: Object
 Configures paging.
 
 ---
-Paging allows the widget to load data in portions instead of loading it at once. To enable paging, set the **paging**.[enabled](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/paging/enabled.md '{basewidgetpath}/Configuration/paging/#enabled') option to **true**.
+Paging allows the widget to load data in portions instead of loading it simultaneously. To enable paging, set the **paging**.[enabled](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/paging/enabled.md '{basewidgetpath}/Configuration/paging/#enabled') option to **true**.
+
+Users can switch between pages and change paging settings via the [pager](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/pager '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/'), or they can scroll the pages. Paging settings apply with any [scrolling mode](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/scrolling/mode.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/scrolling/#mode').
```
Comment (arminal) [inline api-reference/10 UI Widgets/GridBase/1 Configuration/paging/paging.md]:
```suggestion
Users can switch between pages and change paging settings using the [pager](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/pager '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/') or they can scroll the pages. Paging settings apply with any [scrolling mode](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/scrolling/mode.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/scrolling/#mode').
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/scrolling/mode.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ Each next page is loaded once the scrollbar reaches the end of its scale. In thi
...
     
 - **Virtual**       
-Rows are loaded when they get into the viewport and removed once they leave. If the rows take time to be loaded and rendered, they display gray boxes. [Rendering optimization](/Documentation/Guide/Widgets/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization) can reduce rendering time and remove gray boxes. In this mode, users can move to any page instantly.
+Rows are loaded when they get into the viewport and removed once they leave. If the rows take time to be loaded and rendered, they display gray boxes. [Rendering optimization](/concepts/05%20Widgets/DataGrid/10%20Enhance%20Performance%20on%20Large%20Datasets/030%20Rendering%20Optimization.md '/Documentation/Guide/Widgets/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization') can reduce rendering time and remove gray boxes. In this mode, users can move to any page instantly.
```
Comment (arminal) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/scrolling/mode.md]: leave?

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/scrolling/mode.md [hunk 3]
```diff
@@ -20,7 +20,9 @@ Each next page is loaded once the scrollbar reaches the end of its scale. In thi
...
+
+Regardless of the scrolling mode, you can use the **paging**.[pageSize](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/paging/pageSize.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize') option to specify the number of rows in a page.
```
Comment (arminal) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/scrolling/mode.md]:
```suggestion
Regardless of the scrolling mode, you can use the **paging**.[pageSize](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/paging/pageSize.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize') option to specify the number of rows on a page.
```

File: api-reference/10 UI Widgets/dxTreeList/1 Configuration/paging/paging.md [hunk 4]
```diff
@@ -7,7 +7,9 @@ type: Object
 Configures paging.
 
 ---
-Paging allows the widget to render rows by pages instead of rendering them at once. To enable paging, set the **paging**.[enabled](/api-reference/10%20UI%20Widgets/dxTreeList/1%20Configuration/paging/enabled.md '{basewidgetpath}/Configuration/paging/#enabled') option to **true**.
+Paging allows the widget to render rows by pages instead of rendering them simultaneously. To enable paging, set the **paging**.[enabled](/api-reference/10%20UI%20Widgets/dxTreeList/1%20Configuration/paging/enabled.md '{basewidgetpath}/Configuration/paging/#enabled') option to **true**.
+
+Users can switch between pages and change paging settings via the [pager](/api-reference/10%20UI%20Widgets/dxTreeList/1%20Configuration/pager '/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/pager/'), or they can scroll the pages. Paging settings apply with any [scrolling mode](/api-reference/10%20UI%20Widgets/dxTreeList/1%20Configuration/scrolling/mode.md '/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/scrolling/#mode').
```
Comment (arminal) [inline api-reference/10 UI Widgets/dxTreeList/1 Configuration/paging/paging.md]:
```suggestion
Users can switch between pages and change paging settings using the [pager](/api-reference/10%20UI%20Widgets/dxTreeList/1%20Configuration/pager '/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/pager/') or they can scroll the pages. Paging settings apply with any [scrolling mode](/api-reference/10%20UI%20Widgets/dxTreeList/1%20Configuration/scrolling/mode.md '/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/scrolling/#mode').
```

File: api-reference/10 UI Widgets/dxTreeList/1 Configuration/scrolling/mode.md [hunk 5]
```diff
@@ -17,6 +17,8 @@ Rows are rendered at once or by pages if [paging](/api-reference/10%20UI%20Widge
 - **Virtual**       
 This mode is an alternative to paging where pages are rendered when they get into the viewport and removed once they leave it. Use this mode if a user should be able to scroll data by pages.
 
+Regardless of the scrolling mode, you can use the **paging**.[pageSize](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/paging/pageSize.md '/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/paging/#pageSize') option to specify the number of rows in a page.
```
Comment (arminal) [inline api-reference/10 UI Widgets/dxTreeList/1 Configuration/scrolling/mode.md]:
```suggestion
Regardless of the scrolling mode, you can use the **paging**.[pageSize](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/paging/pageSize.md '/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/paging/#pageSize') option to specify the number of rows on a page.
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/scrolling/mode.md (hunk 2 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Widgets/dxDataGrid/1 Configuration/scrolling/mode.md]:
```suggestion
Rows are loaded when they get into the viewport and removed once they leave it. If the rows take time to be loaded and rendered, they display gray boxes. [Rendering optimization](/concepts/05%20Widgets/DataGrid/10%20Enhance%20Performance%20on%20Large%20Datasets/030%20Rendering%20Optimization.md '/Documentation/Guide/Widgets/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization') can reduce rendering time and remove gray boxes. In this mode, users can move to any page instantly.
```

Comment (arminal) [reply api-reference/10 UI Widgets/dxDataGrid/1 Configuration/scrolling/mode.md]: ok

**Pull request merged**